sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/StringIndexType.java

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.ai.textanalytics.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+/**
+ * Defines values for StringIndexType.
+ */
+public enum StringIndexType {
```

Comment:
has this always been public? idk if I am in one of those generated classes... but hopefully we can hide the `StringIndexType `

Reply:
Yes. It is same as the autogeneated code. Are u saying that we should not explore this to user?

Reply:
Yeah. Across languages we are suggesting to keep this hidden from the user

Reply:
I see. removed.